pkg/ocm/client.go

```diff
@@ -0,0 +1,87 @@
+package ocm
+
+import (
+	"fmt"
+	"time"
+
+	sdkClient "github.com/openshift-online/ocm-sdk-go"
+	"github.com/patrickmn/go-cache"
+)
+
+type Client struct {
+	config     *Config
+	logger     sdkClient.Logger
+	connection *sdkClient.Connection
+
+	Authentication OCMAuthentication
+}
+
+type Config struct {
+	BaseURL      string `envconfig:"OCM_BASE_URL" default:""`
+	ClientID     string `envconfig:"OCM_SERVICE_CLIENT_ID" default:""`
+	ClientSecret string `envconfig:"OCM_SERVICE_CLIENT_SECRET" default:""`
+	SelfToken    string `envconfig:"OCM_SELF_TOKEN" default:""`
+	Debug        bool   `envconfig:"OCM_DEBUG" default:"false"`
+}
+
+func NewClient(config Config) (*Client, error) {
+	// Create a logger that has the debug level enabled:
+	logger, err := sdkClient.NewGoLoggerBuilder().
```

Comment:
why not to use same logger as all other code?

Reply:
just for having a simple way to change to debug level using OCM_DEBUG env var. maybe we can think of adding this the same logger instead indeed @rollandf